src/addons/Confirm/Confirm.js

```diff
@@ -52,8 +54,12 @@ class Confirm extends Component {
   };
 
   render() {
+    let classes = classNames(
+      'sd-confirm',
+      this.props.className,
+    );
     return (
-      <Modal ref='modal' className='small'>
+      <Modal {...this.props} className={classes} ref='modal'>
```

Comment:
We [spread](https://facebook.github.io/react/docs/jsx-spread.html) props and use our merged classes for `className`.

Spreading props is the same as mixing objects.  All the keys/values on `this.props` are now on `Modal`.  Specifically listing prop values will override the spread values.  In this case, even if you pass in `className`, it will be overridden by `classes` (which includes `className`).